packages/cli/src/fingerprint/fingerprintWatchCommand.ts

```diff
@@ -59,6 +59,10 @@ export default class FingerprintWatchCommand {
             errorStack: error.stack,
           },
         });
+    } else if (error.message.includes("EMFILE: too many open files")) {
```

Comment:
This message can be different due to internationalization. You should examine `error.code` instead, for clarity maybe using a utility function such as:
```typescript
function isError(error: unknown, code: string): boolean {
  const err = error as NodeJS.ErrnoException;
  return err.code === code;
}
```

```typescript
} else if (isError(error, 'EMFILE')) {
```

Reply:
Added.  To stay consistent with the approach of examining `error.code`, refactored all other message checks in the indexer and scanner to examine `error.code`.  Since `lstat` seems it wouldn't be internationalized, and there could be other types of `UNKNOWN` messages we'd like to know about, changed the check for `UNKNOWN` to be more precise and check for `lstat`.